server/plugin.go

```diff
@@ -32,6 +32,12 @@ type externalConfig struct {
 
 	// Legacy 1.x Webhook secret
 	Secret string `json:"secret"`
+
+	// What MM users can create subscriptions
+	MattermostSubscriptionPermission string
+
+	// Comma separated list of jira groups with permission. Empty is all.
+	JiraGroupsPermission string
```

Comment:
nit: to be more intention-revealing, suggest `SubscriptionEditorsMattermostUserIds` and `SubscriptionEditorsJiraGroups`